sdk/spring/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/repository/TestRepositoryConfig.java

```diff
@@ -47,6 +51,18 @@ public class TestRepositoryConfig extends AbstractCosmosConfiguration {
     @Value("${cosmos.responseContinuationTokenLimitInKb}")
     private int responseContinuationTokenLimitInKb;
 
+    @Value("${cosmos.diagnosticsThresholds.pointOperationLatencyThreshold}")
+    private int pointOperationLatencyThreshold;
```

Comment:
does this parameters required?

Reply:
Yes in Spring they have to be defined, I have not seen any way to do them optionally but what I put in as the defaults in our code base are the default values Java uses.